dashboards/Experimental/MongoDB_ReplSet_Summary_(New).json

```diff
@@ -141,55 +141,129 @@
   "liveNow": false,
   "panels": [
     {
-      "collapsed": true,
+      "collapsed": false,
       "gridPos": {
         "h": 1,
         "w": 24,
         "x": 0,
         "y": 0
       },
-      "id": 1032,
-      "panels": [
-        {
-          "description": "",
-          "gridPos": {
-            "h": 4,
-            "w": 24,
-            "x": 0,
-            "y": 193
-          },
-          "id": 1034,
-          "options": {
-            "code": {
-              "language": "plaintext",
-              "showLineNumbers": false,
-              "showMiniMap": false
-            },
-            "content": "#### 🛎️ Important\n\nThis Dashboard is an experimental MongoDB Instance Summary dashboard in Percona Monitoring and Management (PMM) that is not a part of the official PMM deployment and might be updated.\nWe ship this Dashboard to obtain feedback from our users. Once we officially release this Dashboard, it will be moved to the appropriate folder. \n\nTo provide feedback on the Dashboard, please use [our](https://forums.percona.com) forum.🙏",
-            "mode": "markdown"
-          },
-          "pluginVersion": "9.2.20",
-          "transparent": true,
-          "type": "text"
-        }
-      ],
-      "title": "⚠️ Disclaimer",
+      "id": 1030,
+      "panels": [],
+      "repeat": "service_name",
+      "repeatDirection": "h",
+      "title": "Overview - $service_name",
       "type": "row"
     },
     {
-      "collapsed": false,
+      "description": "Read more about [Replica Set Member States](https://docs.mongodb.com/manual/reference/replica-states/).",
```

Comment:
@rnovikovP should we use link shortener also for external urls? If yes, could you prepare one for @YashSartanpara1

Reply:
yes,  @YashSartanpara1  let's use https://per.co.na/mongo-repstate  (just created )